qiskit/providers/ibmq/runtime/__init__.py

```diff
@@ -91,6 +91,7 @@
 
     from qiskit import IBMQ, QuantumCircuit
     from qiskit.providers.ibmq import RunnerResult
+    from qiskit.providers.ibmq.runtime import ProviderRequestParams as Info
```

Comment:
Using a completely different name for `as` could be very confusing to our users, who are generally less experienced in programming.

Reply:
This is updated -- the example usages should still be revisited though. I'm not sure I understand how the pieces fit together.